src/main/webapp/pipe.js

```diff
@@ -371,6 +374,32 @@ function triggerRebuild(taskId, project, buildId) {
     });
 }
 
+function triggerBuild(taskId) {
+    Q("#rebuild-" + taskId).hide();
+
+    var before;
+    if (crumb.value != null && crumb.value != "") {
+        console.info("Crumb found and will be added to request header");
+        before = function(xhr){xhr.setRequestHeader(crumb.fieldName, crumb.value);}
+    } else {
+        console.info("Crumb not needed");
+        before = function(xhr){}
+    }
+
+    Q.ajax({
+        url: rootURL + "/job/" + taskId + '/build?delay=0sec',
```

Comment:
@marcusphi Yes we should. Right now we dont have any tests for the GUI. Have done some basic Selenium tests here: https://github.com/patbos/delivery-pipeline-plugin/tree/function_test.